peer/peer.go

```diff
@@ -7,11 +7,10 @@ import (
 
 // Peer reperesents a GlusterD
 type Peer struct {
-	ID        uuid.UUID `json:"id"`
-	Name      string    `json:"name"`
-	Addresses []string  `json:"addresses"`
-	Client    bool      `json:"client"`
-	MemberID  uint64    `json:"memberID"`
+	ID       uuid.UUID `json:"id"`
+	Name     string    `json:"name"`
+	Address  string    `json:"address"`
```

Comment:
> But a peer could have multiple addresses

True. I was wondering if there was any real use case on how the multiple peer addresses will be used. I could not think of any. Despite backed by central store, the peers are never stateless (contain bricks), so peer addresses cannot be used for failover or load balancing.